src/pages/Vote/Governance/index.tsx

```diff
@@ -1,19 +1,95 @@
 /** @jsxImportSource @emotion/react */
-import React from 'react';
+import React, { useState } from 'react';
 import { Typography } from '@mui/material';
 import { useTranslation } from 'translation';
+import { useGetProposals } from 'clients/api';
+import { GovernanceProposal, Icon, Spinner, TextButton, Tooltip, Pagination } from 'components';
+import { IProposal } from 'types';
 import { useStyles } from './styles';
 
-export const GovernanceUi: React.FC = () => {
+interface IGovernanceUiProps {
+  proposals: IProposal[];
+  isLoading: boolean;
+  total: number | undefined;
+  limit: number;
+  setCurrentPage: (page: number) => void;
+}
+
+export const GovernanceUi: React.FC<IGovernanceUiProps> = ({
+  proposals,
+  isLoading,
+  total,
+  limit,
+  setCurrentPage,
+}) => {
   const { t } = useTranslation();
   const styles = useStyles();
   return (
     <div css={styles.root}>
-      <Typography variant="h4">{t('vote.governanceProposals')}</Typography>
+      <div css={[styles.header, styles.bottomSpace]}>
+        <Typography variant="h4">{t('vote.governanceProposals')}</Typography>
+        <div css={styles.createProposal}>
+          <TextButton css={styles.marginless}>{t('vote.createProposal')}</TextButton>
+          <Tooltip title={t('vote.requiredVotingPower')} css={styles.infoIcon}>
+            <Icon size="16px" name="info" />
```

Comment:
Ideally the icon size should be defined in the `styles` file through CSS

Reply:
I don't know because for things like size on the icon and color on text I think this is the cleanest option. I don't think we should have a wide prop surface for styling but simple styles that will change frequently and are a primary feature of the component make sense as a prop to me. Writing this as styles ends up being more verbose to apply a single line of css.

What do you think if the value was a number instead of a string with a unit? Otherwise, if we really don't want to use this prop we should remove it

Reply:
My main concern is that doing things like this moves us aways from separating the styles from the DOM. I think it makes sense when it's a variant ("primary", "secondary" etc..) because it represents a separate component in a way, but for granular settings like this one in my opinion it should be defined in the CSS.
I understand it makes things more verbose, but hopefully it won't be needed often because the default width/height set in the `Icon` component is the most used.

And by the way, in this case you can simply remove the `size` prop, because 16px is the default width and height.